Project.toml

```diff
@@ -5,6 +5,8 @@ version = "0.4.0"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
I'm a bit confused as to how Plots snuck into the top level TOML. I'm not sure it needs to be here?

Reply:
@jeremylt, ok B
ut Plots can be in the toml file inside the jupyter directory?

Reply:
Yeah, it should be there. It's there on the main branch